src/usePerpsCommitOrder.ts

```diff
@@ -22,14 +22,14 @@ export function usePerpsCommitOrder({
 }: {
   perpsAccountId?: ethers.BigNumber;
   marketId: string;
-  priceIds?: string[];
   provider?: ethers.providers.Web3Provider;
   walletAddress?: string;
   feedId?: string;
   settlementStrategyId?: string;
   onSuccess: () => void;
 }) {
   const { chainId, queryClient } = useSynthetix();
+  const { data: priceIds } = useAllPriceFeeds();
```

Comment:
We dont need price ids here

For query cache cleaning we can clean multiple caches

Reply:
Because price ids were moved inside getPriceTxn it is now incorrect to get price ids outside of that function and ever use anywhere in the code

Reply:
Removed  `useAllPriceFeeds()`.
Added `usePriceUpdateTxn()`:

`const { data: priceUpdateTxn } = usePriceUpdateTxn({ provider });`

Replaced cache cleaning: 
from:  `queryKey: [chainId, 'PriceUpdateTxn', { priceIds: priceIds?.map((p) => p.slice(0, 8)) }]`
to:  `queryKey: [chainId, 'PriceUpdateTxn']`